### PR TITLE
Logging of trigger in-flight command updates

### PR DESCRIPTION
**Pull request by carlpulley-da**
This PR enables context logging for all updates of the `Runner` in-flight command trie map.

This PR has been cherry picked from https://github.com/digital-asset/daml/pull/15564

- [x] rebase PR once https://github.com/digital-asset/daml/pull/15619 is merged

CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala [hunk 1]
```diff
@@ -579,13 +606,6 @@ private[lf] class Runner private (
     completionSource merge transactionSource merge heartbeatSource
   }
 
-  private def logReceivedMsg(tm: TriggerMsg): Unit = tm match {
```
Comment (carlpulley-da) [inline triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala:582]: In a future PR, `Ctx` based logging on the flow will render this logging useless - removing it now for simplicity

File: triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala [hunk 2]
```diff
@@ -693,11 +712,6 @@ private[lf] class Runner private (
         )
     }
 
-    val logInitialState =
```
Comment (carlpulley-da) [inline triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala:696]: In a future PR, `Ctx` based logging on the flow will render this logging useless - removing it now for simplicity

File: triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala [hunk 3]
```diff
@@ -709,14 +723,14 @@ private[lf] class Runner private (
     val graph = GraphDSL.createGraph(Sink.last[SValue]) { implicit gb => saveLastState =>
       import GraphDSL.Implicits._
 
-      val msgIn = gb add Flow[TriggerMsg].wireTap(logReceivedMsg _)
+      val msgIn = gb add Flow[TriggerMsg]
```
Comment (carlpulley-da) [inline triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala:726]: In a future PR, `Ctx` based logging on the flow will render this logging useless - removing it now for simplicity

File: triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala [hunk 4]
```diff
@@ -9,7 +9,14 @@ import akka.stream.scaladsl._
...
 import com.daml.ledger.api.v1.command_submission_service.SubmitRequest
-import com.daml.ledger.api.v1.commands.{Command, Commands}
+import com.daml.ledger.api.v1.commands.{
+  Command,
+  Commands,
+  CreateAndExerciseCommand,
+  CreateCommand,
+  ExerciseByKeyCommand,
+  ExerciseCommand,
+}
```
Comment (remyhaemmerle-da) [inline triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala:19]: maybe better
```suggestion
import com.daml.ledger.api.v1.commands._
```

File: triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala [hunk 5]
```diff
@@ -367,7 +375,7 @@ private[lf] class Runner private (
     applicationId: ApplicationId,
     parties: TriggerParties,
 )(implicit loggingContext: LoggingContextOf[Trigger]) {
-  import Runner.{SeenMsgs, logger}
+  import Runner.SeenMsgs
```
Comment (remyhaemmerle-da) [inline triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala:378]: maybe we can import this one globally, as we have done for `logger`

**Review by remyhaemmerle-da — APPROVED** [review]
LGTM

**Pull request merged**